llvm_ir_dataset_utils/compile_time_analysis_tools/SLURM_files/batch_main_body.sh

```diff
@@ -0,0 +1,29 @@
+THREADS=24
```

Comment:
Can we delineate where the user-overrideable parameters are (like for different HPC systems)?

At least add a newline between them and the rest of the file.